dev-resources/logback.xml

```diff
@@ -5,11 +5,10 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
-    <logger name="log4j.logger.org.eclipse.jetty.server" level="warn"/>
-    <logger name="log4j.logger.org.eclipse.jetty.util.log" level="warn"/>
+    <logger name="org.apache.http" level="error"/>
```

Comment:
Just curious, were these changes just useful for dev purposes or were they necessary to fix some of the "logging tests" in conjunction with the corresponding bumps to the new logutils that came with the bump to the new tk version?